handler/interface.go

```diff
@@ -0,0 +1,20 @@
+package handler
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-api-clients-go/cantabular"
+	"github.com/ONSdigital/dp-api-clients-go/recipe"
+)
+
+type cantabularClient interface{
+	GetCodebook(context.Context, cantabular.GetCodebookRequest) (*cantabular.GetCodebookResponse, error)
```

Comment:
I think we should also add the healthcheck function in this interface (although it always returns success now, it would be good to have the checkler)
```
Checker(ctx context.Context, state *healthcheck.CheckState) error
```

Reply:
Ah yeah completely forgot you said to implement this, will do.

Reply:
It's already implemented in dp-api-clients-go, we just need to add it to the interface here (and regenerate mocks)

Reply:
Yeah I will need to add it (and for the recipe-api client) but to the service package interface rather than here, as that's where it needs to be used